portfolio/src/main/webapp/style.css

```diff
@@ -116,7 +124,31 @@
   padding: 0 4px;
 }
 
-.fa {
+.fa-google {
+  margin-right:5px;
+  width: 25px;
+  float:left;
+  padding: 10px;
+  font-size: 30px;
+  text-align: center;
+  text-decoration: none;
+  background: #dd4b39;
+  color: white;
+}
+
+.fa-linkedin {
+  margin-right:5px;
+  width: 25px;
+  float:left;
+  padding: 10px;
+  font-size: 30px;
+  text-align: center;
+  text-decoration: none;
+  background: #3B5998;
+  color: white;
+}
+
+.fa-instagram {
```

Comment:
Instagram seems to be unused, were you planning to add it to the html in this PR?

Reply:
fa-instagram is used in the cooking section